### PR TITLE
Adding modifiers to mil std symbols and tactical graphics.

### LINKED ISSUE #179: Can VandV add modifiers to MilStd Symbol feature?
Needs to be defined.

### PR DESCRIPTION
**Pull request by jrummel**
Closes #179

### CODE REVIEW

File: tests/test-vandv/src/main/java/mil/emp3/test/emp3vv/containers/dialogs/AddFeatureDialog.java [hunk 1]
```diff
@@ -13,8 +13,12 @@
 import android.widget.EditText;
 import android.widget.ListView;
 import android.widget.Spinner;
+import android.widget.Toast;
+
+import org.cmapi.primitives.IGeoMilSymbol;
```
Comment (JACochran) [inline tests/test-vandv/src/main/java/mil/emp3/test/emp3vv/containers/dialogs/AddFeatureDialog.java]: are these imports necessary?

File: tests/test-vandv/src/main/java/mil/emp3/test/emp3vv/containers/dialogs/milstdunits/SymbolPropertiesDialog.java [hunk 2]
```diff
@@ -394,6 +410,26 @@ public void onViewCreated(View view, final Bundle savedInstanceState) {
...
+        modifierTypeAdapter = new ArrayAdapter<CharSequence>(this.getContext(), android.R.layout.simple_spinner_item, modifierList);
+        modifierSpinner.setAdapter(modifierTypeAdapter);
+        // Modifier Button
+        Button addModifierButton = (Button) view.findViewById(R.id.add_modifier_button);
+        addModifierButton.setOnClickListener(v -> {
+            modifierMap.put(IGeoMilSymbol.Modifier.valueOf(((String)modifierSpinner.getSelectedItem())), modifierValue.getText().toString());
+            Toast.makeText(this.getContext(), "Added modifier", Toast.LENGTH_SHORT).show();
+        });
+
+
```
Comment (JACochran) [inline tests/test-vandv/src/main/java/mil/emp3/test/emp3vv/containers/dialogs/milstdunits/SymbolPropertiesDialog.java]: remove extra lines (there are 3 could go down to just 1)

**Review by JACochran — DISMISSED** [review]

File: tests/test-vandv/src/main/java/mil/emp3/test/emp3vv/containers/dialogs/AddFeatureDialog.java (hunk 1 above)
Comment (jrummel) [reply tests/test-vandv/src/main/java/mil/emp3/test/emp3vv/containers/dialogs/AddFeatureDialog.java]: Looks like a no. Removed.

File: tests/test-vandv/src/main/java/mil/emp3/test/emp3vv/containers/dialogs/milstdunits/SymbolPropertiesDialog.java (hunk 2 above)
Comment (jrummel) [reply tests/test-vandv/src/main/java/mil/emp3/test/emp3vv/containers/dialogs/milstdunits/SymbolPropertiesDialog.java]: Fixed

**Review by JACochran — APPROVED** [review]

**Pull request merged**